### ISSUE TITLE
multichar overrides in g:targets_aiAI

### ISSUE THREAD

**Issue by sunaku** [wellle/targets.vim#114]
I want to override the default `A` and `I` mappings with `,a` and `,i` respectively.  The current 4-character length limitation on `g:targets_aiAI` prevents me from specifying the mappings I want.

**Comment by wellle** [comment]
Yeah it might actually be possible to do exactly that by using `type()`. I will consider it.
